middleware/cmd/middleware/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"flag"
+	"github.com/digitalbitbox/bitbox-base/middleware/src"
+	"github.com/digitalbitbox/bitbox-base/middleware/src/handlers"
+	"github.com/digitalbitbox/bitbox-base/middleware/util/logging"
+	"github.com/sirupsen/logrus"
+	"net/http"
+)
+
+func main() {
+	bitcoinRpcUser := flag.String("rpcuser", "rpcuser", "Bitcoin rpc user name")
+	bitcoinRpcPassword := flag.String("rpcpassword", "rpcpassword", "Bitcoin rpc password")
+	bitcoinRpcPort := flag.String("rpcport", "8332", "Bitcoin rpc port, localhost is assumed as an address")
+	lightningRpcPath := flag.String("lightning-rpc-path", "/home/bitcoin/.lightning/lightning-rpc", "Path to the lightning rpc unix socket")
+	flag.Parse()
+	logging.Set(&logging.Configuration{Output: "STDERR", Level: logrus.DebugLevel})
+	log := logging.Get().WithGroup("middleware")
+	defer func(log *logrus.Entry) {
```

Comment:
(Optional) It might be more readable to define a variable holding this `func` separately. I.e:

```
// Recover from all panics and log error before panicking again.
logBeforeExit := func(log *logrus.Entry) {
	if r := recover(); r != nil {
		// r is of type interface{} and thus we cannot use log.WithError(r).
		log.WithField("error", r).Error(r)
		panic(r)
}
defer logBeforeExit(log)
```